pg_store_plans.c

```diff
@@ -1472,14 +1473,14 @@ pg_store_plans_internal(FunctionCallInfo fcinfo,
 			values[i++] = Int64GetDatumFast(queryid);
 			values[i++] = Int64GetDatumFast(planid);
 			if (api_version == PGSP_V1_5)
-				values[i++] = ObjectIdGetDatum(queryid);
+				values[i++] = Int64GetDatumFast(queryid);
 		}
 		else
 		{
-			values[i++] = Int64GetDatumFast(0);
-			values[i++] = Int64GetDatumFast(0);
+			values[i++] = Int64GetDatumFast(zero);
```

Comment:
Should the fields be NULL rather than 0?  That's what is being done for pg_stat_statements for instance.

Reply:
Sounds sensitive.  Compatibility won't matter since the field doesn't match a meaningful record of pg_stat_statements.